presidio_evaluator/data_objects.py

```diff
@@ -85,10 +85,9 @@ def from_faker_span(cls, faker_span: FakerSpan) -> "Span":
 
```

Comment:
Changes here:
- sample_id
- formatting
- changed the __repr__ (how the instance is presented when printing it)